objects/LuaScript.ttslua

```diff
@@ -968,6 +969,11 @@ function detectSeatedPlayers()
       if token ~= nil then token.destruct() end
     end
   end
+  for _, guid in ipairs(expansionRulesCopies) do
+    local object = getObjectFromGUID(guid)
+    if object ~= nil then object.destruct() end
+  end
```

Comment:
Why do we have the rules in there in the first place if they always get deleted? Am I missing something?